osbrain/tests/test_bugs.py

```diff
@@ -21,10 +21,6 @@ def inc(agent):
     agent = run_agent('a0')
     agent.set_attr(count=0)
     agent.each(0.0, inc)
-    while True:
-        count = agent.get_attr('count')
-        if count > sys.getrecursionlimit():
-            return
-        time.sleep(0.1)
-        if count == agent.get_attr('count'):
-            raise RuntimeError('Probable recursion level exceeded!')
+
+    limit = sys.getrecursionlimit()
+    assert wait_agent_condition(agent, lambda agent: agent.count > limit)
```

Comment:
Your Windows is slower than Travis... :joy:

What if you change the `time.sleep` to `0.05` instead of `0.01`? Does it change? We could increase the delay instead or as well.